.travis.yml

```diff
@@ -27,6 +28,10 @@ before_install:
   - gem update --system
   - gem update bundler
 
+before_script:
+  # Patch for Ruby 2.4.0 compatibility.
+  - ./scripts/patch_simplecov_0_12_0
+
```

Comment:
I think it would be better to upgrade to `simplecov 0.13`. This issue should be fixed: https://github.com/colszowka/simplecov/blob/0.13.0/CHANGELOG.md